okdata/cli/command.py

```diff
@@ -14,7 +14,15 @@
 
 
 class BaseCommand:
-    __doc__ = f"""usage:
+    version = "0.13.0"
+    # TODO: Can use `importlib` in Python 3.8 and up instead of hard coding the
+    #       version here:
```

Comment:
Burde ikke ha hardcodet i første omgang?Vanskelig å prioritere for å fikse det etterhvert?

Reply:
Vi støtter fortsatt Python 3.7, så kan ikke bruke den nye funksjonen før vi har droppet støtten for det.